onesignalclient/notification.py

```diff
@@ -16,8 +16,7 @@ class Notification():
     IOS_BADGE_TYPE_NONE = 'None'
     IOS_BADGE_TYPE_SETTO = 'SetTo'
     IOS_BADGE_TYPE_INCREASE = 'Increase'
-    IOS_BADGES_TYPES = [
-        IOS_BADGE_TYPE_NONE, IOS_BADGE_TYPE_SETTO, IOS_BADGE_TYPE_INCREASE]
+    IOS_BADGES_TYPES = [IOS_BADGE_TYPE_NONE, IOS_BADGE_TYPE_SETTO, IOS_BADGE_TYPE_INCREASE]
```

Comment:
There's no problems using it this definition like it was before:
```py
    IOS_BADGES_TYPES = [
        IOS_BADGE_TYPE_NONE, IOS_BADGE_TYPE_SETTO, IOS_BADGE_TYPE_INCREASE
    ]
```

Ref.:
`'E501:line too long (91 > 79 characters)'`